### ISSUE TITLE
build/linux-4.14.62/scripts/gcc-version.sh points to wrong musl gcc binary

### ISSUE THREAD

**Issue by sellerie98** [osresearch/heads#619]
The file build/linux-4.14.62/scripts/gcc-version.sh points to the wrong gcc binary:
I cant find the file where the wrong binary is referenced though.

```
make[1]: Entering directory '/media/Storage/git/heads/build/linux-4.14.62/linux-x230'
../scripts/gcc-version.sh: line 26: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: No such file or directory
../scripts/gcc-version.sh: line 27: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: No such file or directory
../scripts/gcc-version.sh: line 29: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: No such file or directory
../scripts/gcc-version.sh: line 26: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: No such file or directory
../scripts/gcc-version.sh: line 27: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: No such file or directory
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
  CHK     include/config/kernel.release
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
  Using .. as source for kernel
make[3]: /media/Storage/git/heads/build/../crossgcc/x86_64-linux-musl/bin/x86_64-musl-linux-gcc: Command not found
Cannot use CONFIG_CC_STACKPROTECTOR_STRONG: -fstack-protector-strong not supported by compiler
make[3]: *** [/media/Storage/git/heads/build/linux-4.14.62/Makefile:1110: prepare-compiler-check] Error 1
make[2]: *** [Makefile:146: sub-make] Error 2
make[1]: *** [Makefile:24: __sub-make] Error 2
make[1]: Leaving directory '/media/Storage/git/heads/build/linux-4.14.62/linux-x230'
make: *** [Makefile:378: /media/Storage/git/heads/build/linux-4.14.62/linux-x230/.build] Error 1
```

Edit:
Found the script that seeminly dynamically assembles the path to the binary, but apparently incorrectly: build/linux-4.14.62/scripts/gcc-version.sh
As I dont understand what is going on in this script I cant fix it.

**Comment by tlaurion** [comment]
@osresearch

**Comment by osresearch** [comment]
It's fixed in https://github.com/osresearch/heads/commit/e5038e6adf9133a51c80b9699e552a68167db63b

That could be cherry picked and applied to master

**Comment by tlaurion** [comment]
@sellerie98  should be fixed with latest commit 8af849c

Please confirm and close ticket

**Comment by tlaurion** [comment]
@sellerie98 please reopen if still problematic
